src/charm.py

```diff
@@ -53,7 +56,7 @@ def __init__(self, *args):
         self._config_path = LocalPath(f"{self._script_exporter_dir}/config.yaml")
         self._binary_path = LocalPath("/usr/local/bin/script_exporter")
         self._binary_resource_name = "script-exporter-binary"
-        self._script_daemon_service = Path("/etc/systemd/system/script-exporter.service")
+        self._script_daemon_service = Path("/etc/systemd/system/{}".format(SERVICE_FILENAME))
```

Comment:
We can use f-strings since it was introduced in python 3.6, and Ubuntu 20.04 runs python 3.8.x

```suggestion
        self._script_daemon_service = Path(f"/etc/systemd/system/{SERVICE_FILENAME}")
```

Reply:
I avoided f-strings because there is some uncertainty about the future of #37 - we may need an 18.04 release.